### PR TITLE
React & Redux group project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by Lucas-Erkana**
- [x] Use React documentation
- [x] Use React components
- [x] Use React props
- [x] Use React Router
- [x] Connect React and Redux
- [x] Handle events in a React app
- [x] Write unit tests with React Testing Library
image

- [x] Use styles in a React app
- [x] Use React hooks
- [x] Apply React best practices and language style guides in code
- [x] Use store, actions and reducers in React
- [x] Perform a code review for a team member

[Live demo](https://astro-ravellers.onrender.com/)

### CODE REVIEW

File: src/components/myprofile.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+              return <li className="data" key={rocket.id}>{rocket.name}</li>;
+            }
+            return '';
+          })}
+        </ul>
+      </div>
+      <div className="container">
+        <h2 className="Mymish">My Missions</h2>
+        <ul className="containers">
+          {missionsData.map((mission) => {
```
Comment (Ghiftee) [inline src/components/myprofile.js]: - [x] Kindly ensure that you use ```filter()``` to render a list of all joined missions and reserved rockets as per the requirements.

File: src/components/missions.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```
Comment (Ghiftee) [inline src/components/missions.js]: - [x] Let's try not to disable linters as they help us write cleaner code.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @Lucash2022 and @stevenmukama  👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ App runs smoothly
✔️ Project is properly documented
✔️ No linter errors
✔️ You used gitflow
However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

- [x] According to the requirements, since you are a team of two, you are not to implement anything for the dragons page. However, I can see you have worked on the dragons' page. I highly commend you for choosing to push yourselves further but in the context of this project, please remove the dragons' page.
- [x] Kindly ensure that you add the requirement in the image below as your project is currently missing it
Screenshot 2023-01-31 at 17 27 58 (2)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- For a better user experience, I suggest you add a placeholder to the profile section for when no missions have been joined or rockets reserved.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/missions.js (hunk 2 above)
Comment (Lucas-Erkana) [reply src/components/missions.js]: Hi @Ghiftee 

removed disable at this [commit](https://github.com/Lucash2022/React-Group-Project/pull/34/commits/f06f06f581e8f5af40747f8576bcf1b55b42106c)

File: src/components/myprofile.js (hunk 1 above)
Comment (stevenmukama) [reply src/components/myprofile.js]: Hi @Ghiftee 
Thank you again for pointing this out.
> * [ ]  Kindly ensure that you use `filter()` to render a list of all joined missions and reserved rockets as per the requirements.
- Use filter() at this [commit](https://github.com/Lucash2022/React-Group-Project/commit/a77c6a699e93ade274cbe3629c5d80579faebc8c)

**Comment by Lucas-Erkana** [comment]
@Ghiftee 

Thank you again for pointing out that we should remove the dragons' page and underline active navigation link. 👍 

We have underlined active navigation at this [commit](https://github.com/Lucash2022/React-Group-Project/pull/34/commits/b4237bb96bc83100b3158b837a297e5a93c9f29f). 💯 

Plus we have also removed dragons' page at this [commit](https://github.com/Lucash2022/React-Group-Project/pull/34/commits/86c8fcb2da4b1729ef12fbdc466016328b199aef) 💯

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,9 @@
...
-    </div>
-  );
-}
+import React from 'react';
+import Navbar from './components/navbar';
 
+const App = () => (
+  <>
+    <Navbar />
+  </>
```
Comment (OLIPLICHE) [inline src/App.js:7]: - okay 👍 
Capture

**Review by OLIPLICHE — APPROVED** [review]
Hi @Lucash2022, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Applied the changes requested by the previous review👏 👏
- The App is working fine👏 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**